### PR TITLE
Add grid 4 section

### PR DESCRIPTION
**Pull request by raju249**
### Description
The new section looks as below:

grid4-test

### Types of changes

- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [ ] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

**Comment by raju249** [comment]
Thanks to @manoj9788 for the new image. 🙏 

@diemol @Harsha509 please take a look. This is the first draft of the new grid 4 section. We can iterate with new stuff in multiple PRs.

Thanks

File: docs_source_files/content/grid4/_index.de.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+zu übersetzen indem Du uns einen Pull Reqeust schickst!
+ {{% /notice %}}
+
+_Selenium Grid 4_ is a smart proxy server
+that allows Selenium tests to route commands to remote web browser instances.
+Its aim is to provide an easy way to run tests in parallel on multiple machines.
+
+With Selenium Grid 4,
+one server acts as the hub that routes JSON formatted test commands
+to one or more registered Grid 4 nodes.
```
Comment (harsha509) [inline docs_source_files/content/grid4/_index.de.md]: Can we remove '4' here. While i read it makes me a bit confusion like `Grid for nodes` and later i get it as Grid 4 as this document is for Grid 4. As we are already mentioning Selenium Grid 4 in title, Can we simply say Grid nodes/ registered nodes ?

File: docs_source_files/content/grid4/_index.en.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Grid 4"
+chapter: true
+weight: 9
+---
+
+# Grid 4
```
Comment (harsha509) [inline docs_source_files/content/grid4/_index.en.md]:
```suggestion
# Selenium Grid 4
```

File: docs_source_files/content/grid4/_index.de.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+Tests contact the hub to obtain access to remote browser instances.
+The hub has a list of registered servers that it provides access to,
+and allows control of these instances.
+
+Selenium Grid 4 allows us to run tests in parallel on multiple machines,
```
Comment (harsha509) [inline docs_source_files/content/grid4/_index.de.md]:
```suggestion
This feature allows us to run tests in parallel on multiple machines,
```

File: docs_source_files/content/grid4/_index.es.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+---
+title: "Grid"
+chapter: true
+weight: 9
+---
+
+# Grid
```
Comment (harsha509) [inline docs_source_files/content/grid4/_index.es.md]:
```suggestion
# Selenium Grid 4
```

Comment (harsha509) [reply docs_source_files/content/grid4/_index.es.md]: The same changes needed for all translated pages

File: docs_source_files/content/grid4/components_of_a_grid.ko.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
+---
+title: "Components of a Grid"
+weight: 2
+---
```
Comment (harsha509) [inline docs_source_files/content/grid4/components_of_a_grid.ko.md]: Are the components of Grid4 is different from legacy grid ? If not we have introduce Grid 4 with in legacy grid document.

We will end up having duplicate documents for `components of grid` and `purposes and functionalities` if there are no changes from previous.

@diemol Can you please suggest on this!

**Review by Harsha509 — DISMISSED** [review]
Hi @raju249 ,

Thanks for the PR.

Also very thanks on taking time to translate few pages in respective languages.

I've requested for few changes, can you please look into it.

Regards,
Harsha.

**Comment by raju249** [comment]
Thanks @Harsha509 for the review.

I have made the other changes, awaiting @diemol's suggestions for [your comment](https://github.com/SeleniumHQ/seleniumhq.github.io/pull/461#discussion_r443139434)

File: docs_source_files/content/grid/grid_4/components_of_a_grid.de.md [hunk 6]
```diff
@@ -1,12 +1,39 @@
...
 
 {{% notice info %}}
-<i class="fas fa-language"></i> Diese Seite wird von Englisch
-auf Deutsch übersetzt. Sprichst Du Deutsch? Hilf uns die Seite
+<i class="fas fa-language"></i> Diese Seite wird von Englisch 
+auf Deutsch übersetzt. Sprichst Du Deutsch? Hilf uns die Seite 
 zu übersetzen indem Du uns einen Pull Reqeust schickst!
-{{% /notice %}}
+ {{% /notice %}}
+![Grid](/images/grid.png)
```
Comment (diemol) [inline docs_source_files/content/grid/grid_4/components_of_a_grid.de.md]: This is copying the same text from the Grid 3 section. This section should be written again for Grid 4.

Comment (raju249) [reply docs_source_files/content/grid/grid_4/components_of_a_grid.de.md]: Hey @diemol yes, I am changing it, I just copied those files. Changing it 👍

**Comment by diemol** [comment]
The image is based on the one presented at the Grid workshop done during the Selenium Conference. I will update the original image and add it to this PR.
image

**Review by diemol — APPROVED** [review]
I removed the image and will merge this. I'll itereate on the content in the next couple of days since I am working on the Grid workshop content.

**Pull request merged**